contracts/core/libraries/Transfer.sol

```diff
@@ -65,9 +65,9 @@ library Transfer {
     function transferAmountFrom(address _token, address _from, address _to, uint256 _amount) internal {
         if (_token == NATIVE) {
             // '_from' is ignored. The contract's balance is used.
-            if (_to != address(this)) _to.safeTransferETH(_amount);
+            if (_to != address(this)) payable(_to).transfer(_amount);
```

Comment:
we should still use safe transfer for native transfers

Reply:
There is no `safeTransferETH` function in OZ library, you mean using both libraries at the same time?